macros/datetime/date_spine.sql

```diff
@@ -60,7 +60,7 @@ filtered as (
 
     select *
     from all_periods
-    where date_{{datepart}} <= {{ end_date }}
+    where date_{{datepart}} <= {{ dbt_utils.safe_cast(end_date, dbt_utils.datatypes.datetime()) }}
```

Comment:
I think we just want to call `dbt_utils.type_datetime()` here, but I love the idea of namespacing macros 😍 

```suggestion
    where date_{{datepart}} <= {{ dbt_utils.safe_cast(end_date, dbt_utils.type_datetime()) }}
```

Reply:
Whoops, that must have been wishful thinking! ;)